spec/spec_join_addfields/section/join-examples.md

```diff
@@ -0,0 +1,254 @@
+## Join examples {#join-examples}
+
+### Joining two CSV files
+The following example illustrates a left join between two csv files. 
+
+**chains.csv**
+<pre class="ex-input">
+"id","title","author"
+"FNAC","The Republic","Plato"
+"FNAC","Moby Dick","Melville"
+"Waterstones","Poetics","Aristotle"
+"Waterstones","Oliver Twist","Dickens"
+</pre>
+**bookstats.csv**
+<pre class="ex-input">
+"id","title","sales"
+"book1","The Republic",12459
+"book2","Moby Dick",335555
+"book3","Poetics",254
+</pre>
+
+<pre class="ex-source">
+<#bookstats> a rml:LogicalSource;
+    rml:source [ 
+        a rml:Source, a csvw:Table;
+        csvw:url "/path/to/bookstats.csv";
+    ];
+    rml:referenceFormulation rml:CSV;
+.
+</pre>
+
+<pre class="ex-source">
+<#chains_with_sales> 
+    rml:source [ 
+        a rml:Source, a csvw:Table;
+        csvw:url "/path/to/chains.csv"
+        ];
+    rml:referenceFormulation ql:CSV;
+    rml:leftJoin [ 
+        a rml:Join ;	 
+        rml:parentLogicalSource <#bookstats> ;
+        rml:joinCondition [
+            rml:parent "title" ;
+            rml:child "title" ;
+        ] ;
+        rml:addField "sales" ;
+        rml:addFieldAlias [
+            rml:parentField "id" ;
+            rml:as "book_id" ;
+        ] ;
+    ] ;
+.
+</pre>
+**Intermediate representation of the logical source <#chains_with_sales>**
+<pre class="ex-intermediate">
+| id          | title        | author    | sales  | book_id |
+|-------------|--------------|-----------|--------|---------|
+| FNAC        | The Republic | Plato     | 12459  | book1   |
+| FNAC        | Moby Dick    | Melville  | 335555 | book2   |
+| Waterstones | Poetics      | Aristotle | 254    | book3   |
+| Waterstones | Oliver Twist | Dickens   |        |         |
+</pre>
+
+
+### Joining a json file with a csv file
+
```

Comment:
I would expect that it should be possible to join a json file with a csv file, under certain restrictions. Up for discussion?

Reply:
Example of this more simple case added.